wiki/markdown/statusCodes.md

```diff
@@ -7,9 +7,9 @@ Status codes are generated according to the below tables.
+|-------------|-------------------------------|-------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 200         | SUCCESS                       | Event sent successfully                                                             | Is returned if the request is completed successfully.                                                                                                                                                                                                                                                                                                                         |
 | 207         | Multi-Status                  |                                                                                     | Is returned if we have a mix of internal status codes.  Eg: publish of few events was successful and few events was a failure.  In case of multiple events in the body JSON, if a event is given internal status code of 400 or 500 rest events will not be published and will be given a internal status code of 503, which will result in overall HTTP response code of 207. |
 | 400         | Bad Request                   | Invalid event content, client need to fix problem in event before submitting again. | Is returned if the request body JSON is malformed. Eg: unable to parse the body JSON.                                                                                                                                                                                                                                                                                          |
 | 404         | RabbitMQ properties not found | RabbitMQ properties not configured for the protocol <protocol>                      | Is returned if RabbitMQ message broker properties are not found for the protocol used by event.                                                                                                                                                                                                                                                                                |
```

Comment:
Status code 415 should be mentioned in this table as well

Reply:
Added